nonlinear-heat_transfer_with_AD_NOX/CMakeLists.txt

```
@@ -0,0 +1,64 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+FIND_PACKAGE(deal.II 9.5.1 QUIET
+  HINTS ${deal.II_DIR} ${DEAL_II_DIR} ../ ../../ $ENV{DEAL_II_DIR}
+  )
+IF(NOT ${deal.II_FOUND})
+  MESSAGE(FATAL_ERROR "\n"
+    "*** Could not locate a (sufficiently recent) version of deal.II. ***\n\n"
+    "You may want to either pass a flag -DDEAL_II_DIR=/path/to/deal.II to cmake\n"
+    "or set an environment variable \"DEAL_II_DIR\" that contains this path."
+    )
+ENDIF()
+
+# Set the variable SOURCE_DIR as ./source, which can be used in the ADD_LIBRARY command
+SET(SOURCE_DIR "./source/")
```

Would you mind setting only the directory name,
```suggestion
SET(SOURCE_DIR "./source")
```
and then adding the slash everywhere below where you use the variable? I think that conforms more with usual styles.